### ISSUE TITLE
Missing Files

### ISSUE THREAD

**Issue by ambs** [srini0x00/dvta#3]
There are missing files in this repo, like Program.cs, and other.

**Comment by RayRRT** [comment]
Yes, seems that some files are missing from the repo, however in the release if all the code is there. 

#Note: Visual studio 2019 locks certain files when compiling, you need to do an "Unblock-File -Path C:\blablablabla" to be able to use those files.
